azure-pipelines.yml

```diff
@@ -441,6 +441,8 @@ stages:
           - checkout: self
             clean: true
           - script: ./eng/cibuild.sh --configuration $(_BuildConfig) --testcoreclr
+            env:
+              COMPlus_DefaultStackSize: 1000000
```

Comment:
This is a way to increase default stack size in macos.
The test failing without it is ~\tests\fsharp\core\quotes\test.fsx
Around line 4000 there is a huge computation expression, compiled as single method.

The IL method size is tens of thousands I believe.

Reply:
I'm wondering if this can hide some an accidental regression introduced by the codegen?

Reply:
If we have regression by codegen which only affects IL size or allocations, we should have explicit tests for that.
This configuration just works arounb MacOS limitation, none of the tests were written in testing stacksize in mind.

Apart from excluding tests selectively on MacOs, I did not have any better idea.